docker-test.yaml

```diff
@@ -92,7 +92,7 @@ services:
       - HOST=${HOST:-localhost}
       - LDAP_DC=DC=AD2012,DC=LAB
       - LOGGING_LEVEL=${LOGGING_LEVEL:-INFO}
-      - NEXT_ADMIN_USER=${NEXT_ADMIN_USER:-Admin}
+      - NEXT_ADMIN_USER=${NEXT_ADMIN_USER:-AdminTest}
```

Comment:
We should set it statically to AdminTest.

Reply:
fixed